### DISCUSSION TITLE
Week 1

### DISCUSSION THREAD

**Discussion by Stev3eM** [SE-TINF22B6/CookHub#7] [Show and tell]
Hello backpackers,

this is our weekly discussion-blog to keep you up to date within our project progression.

For the first week we have a few updates to release:

- setup our team and roles (product owner, scrum master, developer, subtask responsables,...)
- agreed on our project idea of building a cooking help web app 
- agreed on our tech-stack (React, C#, JavaScript/TypeScript, asp.NET) so far
- send all information to our instructor
- booked our e-Portfolio topics
- setup our GitHub wiki and began to fill it
- agreed to use https://tmetric.com/ as our timetracking addon
- setup our project to clone it into IDE

That's everything so far for this week ;-)

Have a nice weekend

Your CookHub Team

**Comment by mirak55** [comment]
You gave a good overview of everything you have already accomplished, maybe you could also write down who has what role in your team. Your Kanban board also looks very good with the different tags, making it easy to understand. I'm excited to see first impressions of your website.
Your CookPit team
